types/src/front/api_handlers/internal/agent_configuration.ts

```diff
@@ -62,7 +62,7 @@ const RetrievalActionConfigurationSchema = t.type({
   dataSources: t.array(
     t.type({
       dataSourceId: t.string,
-      dataSourceViewId: t.union([t.string, t.null]),
+      dataSourceViewId: t.string,
```

Comment:
This will break the assistant builder for users already on the builder when we release.

Reply:
What do you propose ?